_resources/port1.0/group/haskell-stack-1.0.tcl

```diff
@@ -0,0 +1,32 @@
+depends_build-append  \
```

Comment:
Please add the first line shebang to this new file. Also, could you please add a brief description at the top of the port documenting its purpose and basic usage in ports ?

Reply:
Done.